sdk/cosmos/azure-cosmos-kafka-connect/src/docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Build the Cosmos DB Connectors on top of the Kafka Connect image
+FROM confluentinc/cp-kafka-connect:6.0.0
```

Comment:
Should this be pushed to eng\versioning as well?

Reply:
It will be depending on how we are going to setup the Integration pipeline. Using this working item to track it: https://github.com/Azure/azure-sdk-for-java/issues/38770